src/transformers/models/bart/modeling_bart.py

```diff
@@ -253,7 +253,7 @@ def forward(
 
         attn_output = attn_output.view(bsz, self.num_heads, tgt_len, self.head_dim)
         attn_output = attn_output.transpose(1, 2)
-        attn_output = attn_output.reshape(bsz, tgt_len, embed_dim)
+        attn_output = attn_output.reshape(bsz, tgt_len, self.embed_dim)
```

Comment:
if we do that, I propose we change the earlier code:

```
- bsz, tgt_len, embed_dim = hidden_states.size()
+bsz, tgt_len, _ = hidden_states.size()
```
so that there will be no confusing `embed_dim` local variable hanging around.

Reply:
Great.